docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -169,7 +169,7 @@ You can [implement custom converters](system-text-json-converters-how-to.md) to
 
 ## How to read JSON as .NET objects (deserialize)
 
-To deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method.
+A common way to deserialize JSON is to first create a class with properties and fields that represent one or more of the JSON properties. Then, to deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method. You can deserialize either synchronously or asynchronously. Any JSON properties that aren't represented in your class are ignored.
```

Comment:
@tdykstra Is this accurate? I think it adds some nice extra info about how the class is incorporated into the process.

```suggestion
A common way to deserialize JSON is to first create a class with properties and fields that represent one or more of the JSON properties. Then, to deserialize from a string or a file, call the <xref:System.Text.Json.JsonSerializer.Deserialize%2A?displayProperty=nameWithType> method. For the generic overloads, you pass the type of the class you created as the generic type parameter. For the non-generic overloads, you pass the type of the class you created as a method parameter. You can deserialize either synchronously or asynchronously. Any JSON properties that aren't represented in your class are ignored.
```

Reply:
Sounds good.